webrtc-c/canary/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.14)
+project(KVSWebRTCCanary LANGUAGES C CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+
+include(FetchContent)
+
+# AWS SDK options
+set(CUSTOM_MEMORY_MANAGEMENT OFF)
+set(ENABLE_TESTING OFF)
+set(BUILD_ONLY "monitoring;logs")
```

Comment:
I think we should build events as well like we do in producer C. From what I recall, we do use it somewhere in producer C atleast.

Reply:
We can definitely add this when we actually need the API. But, so far, I haven't found the need for it.